ui/app/routes/vault/cluster/dashboard.js

```diff
@@ -0,0 +1,58 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+import Route from '@ember/routing/route';
+import { inject as service } from '@ember/service';
+import { hash } from 'rsvp';
+// eslint-disable-next-line ember/no-mixins
+import ClusterRoute from 'vault/mixins/cluster-route';
+import { action } from '@ember/object';
+
+export default class VaultClusterDashboardRoute extends Route.extend(ClusterRoute) {
+  @service store;
+  @service namespace;
+  @service version;
+
+  async getVaultConfiguration() {
+    try {
+      const adapter = this.store.adapterFor('application');
+      const configState = await adapter.ajax('/v1/sys/config/state/sanitized', 'GET');
+      return configState.data;
+    } catch (e) {
+      return null;
+    }
+  }
+
+  async getLicense() {
+    try {
+      return await this.store.queryRecord('license', {});
+    } catch (e) {
+      return null;
+    }
+  }
+
+  model() {
+    const vaultConfiguration = this.getVaultConfiguration();
```

Comment:
nit: since we're not using `vaultConfiguation` anywhere except on the return hash, we can instead grab it within the has like 
```
return hash({
  vaulConfiguration: this.getVaultConfiguration(),
  ...
})
```